### ISSUE TITLE
Improve keyboard navigation and add hotkeys for formatting

### ISSUE THREAD

**Issue by Feltzer** [byhestia/springseed#47]
I recently stumbled on Springseed and it quickly became my favorite note-taking app on Ubuntu.

The only department I feel it's lacking in is keyboard navigation. For instance, I would love to be able to:
- quickly move between notebooks and notes with TAB and the arrow keys.
- toggle the editing mode with a hotkey like CTRL+E
- save notes with CTRL+S
- apply Markdown formatting with CTRL+B (bold), CTRL+I (italics), CTRL+U (URL), CTRL+H (Heading↑)

These hotkeys are only quickly thought up suggestions, of course. I would be very glad if you could give implementing any of these keyboard enhancements a thought.

Thanks for reading this.
